classes/sota.bbclass

```diff
@@ -46,11 +46,11 @@ GARAGE_TARGET_EXPIRE_AFTER ?= ""
 GARAGE_CUSTOMIZE_TARGET ?= ""
 
 SOTA_MACHINE ??="none"
-SOTA_MACHINE_rpi ?= "raspberrypi"
+SOTA_MACHINE:rpi ?= "raspberrypi"
 SOTA_MACHINE_porter ?= "porter"
 SOTA_MACHINE_m3ulcb = "m3ulcb"
 SOTA_MACHINE_intel-corei7-64 ?= "minnowboard"
-SOTA_MACHINE_qemux86-64 ?= "qemux86-64"
+SOTA_MACHINE:qemux86-64 ?= "qemux86-64"
 SOTA_MACHINE_am335x-evm ?= "am335x-evm-wifi"
 SOTA_MACHINE_freedom-u540 ?= "freedom-u540"
```

Comment:
Seems like some items here are skipped.

Reply:
Yes, the list of possible overrides is hardcoded in the conversion script, so these should be migrated as well.